chatty-clownfish/bangazon-sprint-2
# Navbar

-Added navbar.html and base.html
-Wrapped existing templates in base.html to populate navbar
-Added 2 urls for profile and cart -- currently commented out to not break site

TO TEST:

Make sure navbar exists on extensions:  /sell, /products,  and /  (navbar is not on register or login pages)
Nav links to home, my products, and login should work
Once user is logged in, the 'login' link should change to 'logout'
The logout link should log the user out and redirect to the home screen
Once user is logged out, the nav button should change back to 'login'
